hamp_radar/iquick.py

```diff
@@ -222,26 +399,49 @@ def read_iq(filename, postprocess=True):
 
 
 def decode_time(ds):
+    """
+    Replaces 'Tm' and 'time_milli' variables in dataset 'ds' with decoded time.
+
+    The function calculates the time by adding:
+    - A base time of "1970-01-01"
+    - 'Tm' multiplied by 1 second (expressed as 1,000,000,000 ns)
+    - 'time_milli' multiplied by 0.001 (expressed as ns)
```

Comment:
This text block is basically a verbatim copy of the code below, and also not helpful for outside users.
```suggestion
```